src/api/notes/dto-methods.ts

```diff
@@ -7,11 +7,25 @@
 import { NoteDto } from './types'
 import { NoteDetails } from '../../redux/note-details/types'
 import { DateTime } from 'luxon'
-import { initialState } from '../../redux/note-details/reducers'
+import { initialState } from '../../redux/note-details/initial-state'
 
+/**
+ * Converts a note DTO from the API to a {@link NoteDetails} object.
+ * Note that the documentContent will be set but the markdownContent and rawFrontmatterContent are empty.
+ * Therefore, after getting a note from the API, you need to call the {@link setNoteContent} method.
```

Comment:
If this method is used only in the reducer then  maybe we should change it's purpose from a generic "convert" to "createNoteDetailsFromNoteDto"?

Also: If the method is used only in the reducer you can use the util methods to fill "markdownCntent" and "frontmatter"